core/vm/evm.go

```diff
@@ -265,7 +271,7 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 	return ret, gas, err
 }
 
-// This allows the user transfer balance of a specified coinId in addition to a normal Call().
+// CallExpert allows the user transfer balance of a specified coinId in addition to a normal Call().
```

Comment:
This call gives me the spooks. Probably going to want to deprecate it past the hardfork, but we'll still need to support it for verifying the transitions pre-hardfork.

Reply:
Updated to be deprecated after the hard fork.